lib/google_maps.ex

```diff
@@ -851,6 +851,112 @@ defmodule GoogleMaps do
     GoogleMaps.get("place/queryautocomplete", params)
   end
 
+@doc """
+  Search for nearby places based on location and keywords
+
+  `nearby_search/3/4` - Most common use for Places API.
```

Comment:
While it's totally fine to use the endpoint name as the function name, I would like to think about using the `place` prefix for these function, i.e., `place_nearby`. This is also to follow the convention of `place_query`.